cmd/start.go

```diff
@@ -29,10 +29,20 @@ func init() {
 }
 
 func runStart(cmd *cobra.Command, args []string) {
+	checkDaemon()
 	handleGlobalNetwork()
 	startContainers(startFlags.Services)
 }
 
+func checkDaemon() {
+	_, err := shellExec("docker", "info")
```

Comment:
I understand possibly the `docker info` is something that passes/fails quickly enough. Just need to double check - under Linux, if Docker Daemon is not running, will this indeed error out?

Reply:
Now, a second point - other than checking `docker info`, before it I would like to do two less expansive checks that are great for most use cases we will have - let's use `exec.LookPath` to confirm that both `docker` and `docker-compose` exist and are available for execution in the current PATH settings.